File: Maxima.Domain/Pedido.cs
using System.Collections.Generic;

namespace Maxima.Domain
{
    public class Pedido
    {
        public int PedidoId { get; set; }
        public decimal ValorTotal { get; set; }
        public int QtdeItem { get; set; }

        public decimal ValorFrete { get; set; }

        
        public int ClienteId { get; set; }       
        public Cliente Cliente { get;}

         public int ProdutoId { get; set; }
        public Produto Produto { get;}

        public List<ProdutosClientesPedido> ProdutosClientesPedidos {get;}



    }
}

File: Maxima-App/src/app/_models/Pedido.ts
import { Produto } from './Produto';

export interface Pedido {
  pedidoId: number;
  valorTotal: number;
  qtdeItem: number;
  valorFrete: number;
  clienteId: number;
  produtoId: number;
  produtosClientesPedido: Produto[];
}
